src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -36,7 +35,7 @@ public void visitFirstJohnsDashboard() {
 
 	@And("User clicks on Conditions")
 	public void launchManageConditions() {
-		elementClickOn(ConditionsPage.MANAGE_CONDITIONS_ENDPOINT);
+		dashboardPage.clickOnConditionsWidgetLink();
```

Comment:
this is nice @achilep could you split these changes into another PR to make this PR more readable